x/data/msg_anchor_test.go

```diff
@@ -0,0 +1,43 @@
+package data
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/regen-network/gocuke"
+	"github.com/stretchr/testify/require"
+)
+
+type msgAnchorSuite struct {
+	t   gocuke.TestingT
+	msg *MsgAnchor
+	err error
+}
+
+func TestMsgAnchor(t *testing.T) {
+	runner := gocuke.NewRunner(t, &msgAnchorSuite{}).Path("./features/msg_anchor.feature")
+	runner.Step(`a message of "((?:[^\"]|\")*)"`, (*msgAnchorSuite).AMessageOf)
+	runner.Run()
+}
+
+func (s *msgAnchorSuite) Before(t gocuke.TestingT) {
+	s.t = t
+	s.msg = &MsgAnchor{}
+}
+
+func (s *msgAnchorSuite) AMessageOf(a gocuke.DocString) {
+	err := json.Unmarshal([]byte(a.Content), &s.msg)
```

Comment:
we should be using https://pkg.go.dev/github.com/gogo/protobuf/jsonpb

Reply:
now using jsonpb